src/test/java/lotto/model/CashierTest.java

```diff
@@ -0,0 +1,29 @@
+package lotto.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class CashierTest {
+
+    @DisplayName("구입 금액 단위가 일치할 경우, 로또가 정상 발급된다.")
+    @Test
+    void ok() {
+        Cashier cashier = new Cashier();
+        List<Lotto> lottos = cashier.payForLotto(1_000);
+        assertEquals(1, lottos.size());
```

Comment:
Junit 의 `Assertions` 를 사용해서 검증부를 작성했는데 `AssertJ` 도 활용해보면 좋을 것 같아요~
둘의 차이도 비교해보구요 😄 

```suggestion
        assertThat(lottos.size()).hasSize(1);
```

Reply:
저도 `AssertJ`를 사용해서 검증 부분을 작성해봤었는데, 개인적으로 차이점은 

- `AssertJ` : 메서드 이름만으로도 검증으로부터 기대하는 결과를 파악하기 쉽다.
- `JUnit` : 메서드 체이닝을 통해 (실행 결과) -> (기댓값) 흐름으로 검증의 흐름을 알 수 있다.

크게는 이 정도로 생각하고 있어요! 아직 테스트 라이브러리를 많이 못 다뤄봐서 더 큰 차이점을 느끼지 못한 것인지..! 찰리는 어떤 걸 선호하시는지도 궁금해요!

Reply:
`AssertJ`와 `Junit`의 설명이 반대로 된것 같은데 맞을까요? ㅎㅎ 🤔 
저는 `AssertJ` 를 자주 사용하고 있어요
개인적으로는 메서드 이름도 명확하고 여러가지를 제공하구요 
메서드 체이닝으로 검증을 다양하게 진행할 수 있어서요~

```java
assertThatThrownBy(() -> doSomething())
                .isInstanceOf(IllegalArgumentException.class)
                .hasMessage("메세지입니다");
```

Reply:
> `AssertJ`와 `Junit`의 설명이 반대로 된것 같은데 맞을까요? ㅎㅎ 🤔 저는 `AssertJ` 를 자주 사용하고 있어요 개인적으로는 메서드 이름도 명확하고 여러가지를 제공하구요 메서드 체이닝으로 검증을 다양하게 진행할 수 있어서요~
> 
> ```java
> assertThatThrownBy(() -> doSomething())
>                 .isInstanceOf(IllegalArgumentException.class)
>                 .hasMessage("메세지입니다");
> ```

헉 반대로 설명된 것 맞네요 🫢 맞아요 체이닝의 장점이 큰 것 같아요!